src/main/java/edu/oregonstate/cs361/battleship/BattleshipModel.java

```diff
@@ -189,7 +193,7 @@ public boolean goodShot(Coordinate toCheck){
     }
 
     void playerShot(Coordinate coor) {
-        if(playerMisses.contains(coor)){
+        if(playerMisses.contains(coor) || playerHits.contains(coor)){
```

Comment:
Good thinking on this; it should make debugging easier if one of our functions fires on a hit square for some reason.